tests/testthat/test-utils.R

```diff
@@ -125,6 +125,50 @@ test_that("wb_dims() works when not supplying `x`.", {
   expect_error(wb_dims(1, 1, row_names = FALSE), "`row_names`")
 })
 
+test_that("wb_dims() works with above and one line", {
+  expect_equal(wb_dims(x = names(mtcars), from_row = 2), "A2:A12")
+
+  # doesn't work
+  # expect_equal(wb_dims(x = names(mtcars), from_row = 3, above = 1), "A2:A12")
+
+  expect_warning(
+    expect_equal(
+      wb_dims(x = names(mtcars), from_row = 2, above = 1),
+      "A1:A11"
+    ),
+    "rows cannot be above of row 1 "
+  )
+
+  expect_warning(
+    expect_equal(
+      wb_dims(x = names(mtcars), from_dims = "A2", above = 1),
+      "A1:A11"),
+    "rows cannot be above of row 1 "
+  )
```

Comment:
I still don't understand this :( 


I feel like if I read this literally. Let's use a simple example. 

```r
wb_dims(from_row = 3)
#> "A3"
# I feel like reading this literally, we just want the cell above.
wb_dims(from_row = 3, above = 1)
# I would expect this to be "A2"
```

Sorry for being slow here.

Reply:
The vector size is `length(names(mtcars))`. Above 1 tries to add this vector on top of `“A2“`. But there’s only place for a single value. Above, below, left and right simply don’t move integer rows in any direction, but in this case `dim(x)` wise. I tried to make it easier to move around, without having to add the object size multiple times.

If the integer passed to above is greater than 1, the distance between from_dim and the new dim object will be 1 row. I can create a more detailed example later. I guess it’s easier to understand, if you have enough space to the left and right. Maybe start in `"M40"`.


You are simply expecting a different behavior. I didn’t want to move the dims region row or column wise over the sheet.

Reply:
Haha, I hope you tell me if it makes no sense. I have added an example to discussion #1108 , hopefully this makes it clearer what I was intending. What I was striving for was minimizing position calculations. Because otherwise every time when the size of a data frame changes there was a chance of the calculation to be wrong.